_includes/themes/zeppelin/_navigation.html

```diff
@@ -70,9 +70,13 @@
     <p>A web-based notebook that enables interactive data analytics. <br/>
       You can make beautiful data-driven, interactive and collaborative documents with SQL, Scala and more.
     </p>
-    <p><a href="http://youtu.be/_PQbVH_aO5E" target="_zeppelinVideo" class="btn btn-primary btn-lg bigFingerButton" role="button">Watch the video</a>
-
-       <a href="./download.html" class="btn btn-primary btn-lg bigFingerButton" role="button">Get Zeppelin</a></p>
+    <div class="border row">
+      <div class="col-md-12 col-sm-12 col-xs-12 text-center newBtn">
+        <a href="http://youtu.be/_PQbVH_aO5E" target="_zeppelinVideo" class="btn btn-primary btn-lg bigFingerButton" role="button">Tutorial Video</a>
+        <a href="./download.html" class="btn btn-primary btn-lg bigFingerButton" role="button">GET ZEPPELIN 0.7</a>
```

Comment:
this is nice - should it just say "GET LATEST ZEPPELIN NOW"? so we don't have to keep updating this ;)